packages/fxa-payments-server/src/lib/config.ts

```diff
@@ -0,0 +1,53 @@
+// This configuration is a subset of the configuration declared in server/config/index.js
+// Which config is copied over is defined in server/lib/server.js
+const config = {
+  featureFlags: {},
+  servers: {
+    auth: {
+      url: '',
+    },
+    content: {
+      url: '',
+    },
+    oauth: {
+      url: '',
+    },
+    profile: {
+      url: '',
+    },
+  },
+  stripe: {
+    apiKey: '',
+  },
+  lang: '',
+};
+
+function decodeConfig(content: string|null) {
+  if (!content) {
+    throw new Error('Configuration is empty');
+  }
+  const decoded = decodeURIComponent(content);
+  try {
+    return JSON.parse(decoded);
+  } catch (e) {
+    throw new Error(`Invalid configuration ${JSON.stringify(content)}: ${decoded}`);
+  }
+}
+
+function init() {
+  const configEl = document.head.querySelector('meta[name="fxa-config"]');
+  if (!configEl) {
+    throw new Error('<meta name="fxa-config"> is missing');
+  }
+  Object.assign(config, decodeConfig(configEl.getAttribute('content')));
+  const featureEl = document.head.querySelector('meta[name="fxa-feature-flags"]');
+  if (!featureEl) {
+    throw new Error('<meta name="fxa-feature-flags"> is missing');
+  }
+  config.featureFlags = decodeConfig(featureEl.getAttribute('content'));
+  config.lang = document.documentElement.lang;
+}
+
+init();
```

Comment:
Might do with a little control inversion, here.

Instead of calling `init()` here, I'd export it and call it in [the main `index.tsx`](https://github.com/mozilla/fxa/blob/master/packages/fxa-payments-server/src/index.tsx#L24) just before the first `<App>` render.

Also a good idea to accept `document` as a parameter in `init()`. The call in `index.tsx` can pass the real one, but we could easily pass in a mock `document` in unit tests of `config.ts`.

Then, I'd add & export another function for use in tests & stories for setting mock config. Something like:
```javascript
export function setConfig(newConfig) {
  return Object.assign(config, newConfig);
}
```